witms/spiders/foxnews.py

```diff
@@ -9,19 +9,23 @@ class FoxNewsSpider(Spider):
     portal_name = "Fox News"
     allowed_domains = ["foxnews.com"]
     start_urls = ["https://www.foxnews.com"]
-    link_extractor = LinkExtractor(deny=["/v/", "/person/", "video.foxnews.com"])
+    link_extractor = LinkExtractor(
+        deny=["/v/", "/person/", "/category/", "video.foxnews.com"]
+    )
 
     def parse(self, response):
         loader = ArticleLoader(item=Article(), response=response)
         loader.add_value("url", response.url)
         loader.add_value("portal", FoxNewsSpider.portal_name)
         loader.add_xpath("section", '//meta[@name="prism.section"]/@content')
```

Comment:
prism, as in https://en.wikipedia.org/wiki/PRISM_%28surveillance_program%29 ? xD